node/src/main/scala/coop/rchain/node/configuration/Configuration.scala

```diff
@@ -66,10 +71,22 @@ object Configuration {
         configFile <- Task.delay(options.configFile.getOrElse(dataDir.resolve("rnode.toml")).toFile)
         _          <- log.info(s"Using configuration file: $configFile")
         configE    <- Task.delay(toml.TomlConfiguration.from(configFile))
-        _ <- configE match {
-              case Right(_) => Task.unit
-              case Left(e)  => log.warn(s"Can't load the configuration file: $e")
-            }
+        exit <- configE match {
```

Comment:
```scala
configE.leftMap {
                 case Left(ConfigurationParseError(e)) => "Can't parse the configuration: $e"
                 case Left(ConfigurationAstError(e)) =>s"The structure of the configuration is not valid: $e"
                 case Left(ConfigurationFileNotFound(f)) =>s"Configuration file $f not found"
}.fold(Log[F].error(_).as(true), Task.now(false))
```

Reply:
@rabbitonweb ok, but I want to return `false` in case of `ConfigurationFileNotFound`